apps/stable_diffusion/shark_sd.spec

```diff
@@ -42,7 +42,12 @@ datas += [
          ( 'web/ui/logos/*', 'logos' )
          ]
 
+torch_path = sys.prefix + '/lib/python3.11/site-packages/torch/lib/libtorch_python.dylib'
```

Comment:
This won't work on Windows. I think you can just add to copy_data()

Reply:
the spec file already has copy_metadata('torch')